app/assets/javascripts/app/users/user.js

```diff
@@ -5,36 +5,40 @@
     .module('council.users')
     .controller('EditUserCtrl', EditUserController);
 
-  function EditUserController(User, userId, DS) {
+  function EditUserController($state, User, userId) {
```

Comment:
I moved the submit and validation to the form's controller. How does it look now?